### DISCUSSION TITLE
Laptop is screen black but not off when using second screen in Ultimate

### DISCUSSION THREAD

**Discussion by kohanyirobert** [seerge/g-helper#3137] [Q&A]
Hey, I've noticed recently that when I switch to Ultimate and restart my laptop when my second screen is connected when my system is booted my laptop screen is black but it's not off.

I have to switch to _Extend_ and then again to _Second screen only_ to actually turn it off.

Not sure if this is related to G-Helper at all at this point (never used anything else).

My machine is a ROG Zephyrus G14 GA402RK, Windows 11 latest stable updates installed.

Any idea? Minor annoyance, but still.

On thing I haven't tested _yet_ but will do after I write this (just came to me) is that I connect my external display to a USB-C hub instead of directly into the HDMI port.

**Comment by seerge** [comment]
@kohanyirobert it looks like a AMD driver hiccup. G-Helper can't and doesn't manage those things :) I have same device and use external monitor (connected via USB-C tho) and don't experience that. 

Try to do a fresh [AMD driver install ](https://github.com/seerge/g-helper/wiki/Troubleshooting#reinstalling-gpu-drivers) may be ?

**Reply by kohanyirobert** [reply]
Hmm, I just tested it again. Switching between Standard and Ultimate and it happened again.
When connected to HDMI port it didn't. Well, it's late, maybe I'll retest another time.

About the driver. I have Adrenalin installed and this is the driver version(s) I see
Adrenalin version: 24.8.1
Driver version: 24.10.37.04-240823a-407010C-AMD-Software-Adrenalin-Edition
AMD Windows Driver Version: 32.0.11037.4004

Just based on the installer's filename you linked to: `amd-software-adrenalin-edition-24.1.1-combined-minimalsetup-240122_web`
this is Adrenalin version 24.1.1?

Not sure. I'm not fond of Adrenalin, so maybe I'll try this one. Do I understand correctly this download is just the drivers and no Adrenalin junk, right? (I was searching for something like this, but ... didn't find it or maybe read somewhere that the "minimal driver" is lagging behind in version numbers and the best is to use Adrenalin as it is. Thanks for answer.)

**Reply by seerge** [reply]
I just linked an official AMD website https://www.amd.com/en/support/download/drivers.html :) It's not about version, but rather about resetting AMD driver settings to Factory Defaults during install. 

You can select what components to install there ofc, either pure driver or also Adrenaline app next to it.

**Reply by kohanyirobert** [reply]
> It's not about version, but rather about resetting AMD driver settings to Factory Defaults during install.
Did a factory reset, reinstalled the drivers only, no change.

I thought that maybe it's a version thing. Would you be so kind as to share the driver version you use? If it's the latest for you too than something else is happening on my end.

The driver version for the iGPU and dGPU in device manager that I see is: 32.0.11037.4004 (I have only installed the drivers this time).

**Reply by seerge** [reply]
@kohanyirobert i use same driver as you mentioned, 24.8.1. But I did a full install including Adrenaline as i need it to toggle frame-gen thing :)

**Reply by kohanyirobert** [reply]
I had the full version previously so ... my bet is I have some weird Windows 11 setting flipped or similar. ... Maybe Windows 11 in itself a problem :D
